tests/test-extensions/some-extension/src/main/java/org/mule/test/some/extension/SomeEmittingSource.java

```diff
@@ -44,14 +45,19 @@
   Integer times;
   private Scheduler emitterScheduler;
 
+  @ParameterGroup(name = "Awesome Parameter Group")
+  SomeParameterGroupOneRequiredConfig someParameterGroup;
```

Comment:
Won't this make you change al tests that use this source?

Reply:
yeah, already changed the 2 xml files within the mule project that used it. Could not find any usages of this test extension source outside of it.

Reply:
I would create a new source that extends from this one and has this parameter group.

Reply:
changed